### DISCUSSION TITLE
Why aren't new releases posted on github anymore?

### DISCUSSION THREAD

**Discussion by Papela** [YimMenu/YimMenu#1940] [Q&A] [answered]
I have noticed and the last release that exists is from March, but I have found functional versions on the internet.
And if I compile the project I get a new and updated version.
Why don't new versions come out on github anymore but in other places yes?

**Comment by Yimura** [comment, marked as answer]
Check the date of the file upload not the date of the release.

**Comment by Papela** [comment]
Wow, I'm stupid. Thank you
